test/samples/arr_sum_custom_tag.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.8.4;
+
+contract Foo {
+    // Simple map
+    uint[] a = [10];
+    int8[] b;
+
+    /// checks for unchecked_sum() @custom:scribble #if_succeeds unchecked_sum(a) > 10 && unchecked_sum(a) < 20;
+    function pushA(uint k) public {
+        a.push(k);
+    }
+
+    /// checks for unchecked_sum() @custom:scribble #if_succeeds unchecked_sum(a) > 10 && unchecked_sum(a) < 20;
+    function setA(uint k, uint v) public {
+        a[k] = v;
+    }
+
+    /// checks for unchecked_sum() @custom:scribble #if_succeeds unchecked_sum(b) > -10 && unchecked_sum(b) < 10;
+    function pushB(int8 k) public {
+        b.push(k);
+    }
+
+    /// checks for unchecked_sum() @custom:scribble #if_succeeds unchecked_sum(b) > -10 && unchecked_sum(b) < 10;
```

Comment:
Hmm, these shouldn't parse as the `@custom:scribble` tag. Are these being parsed as that?

Reply:
These get parsed in `@custom:scribble` when I used --devdoc.

Reply:
I changed the behavior to ignore any lines that have text before `@custom:scribble` on the same line(05abb16). In a followup PR I will add some extra code to issue warnings when something that looks like an invariant has text before that.